server/src/controllers/eventController.service.ts

```diff
@@ -311,6 +311,7 @@ const validateContact = (formData: checkoutForm) => {
           new RegExp('^(\\+?\\d{1,2}\\s?)?\\(?\\d{3}\\)?[\\s.-]?\\d{3}[\\s.-]?\\d{4}$'),
       ),
     }),
+    ...(formData.visitSource && {visitSource: formData.visitSource}),
```

Comment:
The new column name in `contacts` table is `visitsource` with a lowercase 's'. Updating this line to `...(formData.visitSource && {visitsource: formData.visitSource}),` should get things working